### DISCUSSION TITLE
Criterion error

### DISCUSSION THREAD

**Discussion by David2261** [community/community#37082] [Other Feature Feedback, Questions, & Ideas]
Information blurs when scrolling through the page, on Samsung A21S
![Screenshot_20221024-102726_Chrome](https://user-images.githubusercontent.com/65621634/197471855-f7469a44-b60d-4431-ac2c-a2dfe668a2b8.jpg)

**Comment by ossanna16** [comment]
Hi there @David2261 and welcome to our community! Thank you for reporting this 🙂
